monai/deploy/core/executors/single_process_executor.py

```diff
@@ -36,11 +36,11 @@ class SingleProcessExecutor(Executor):
     def run(self):
         """Run the app.
 
-        This method retrieves the root nodes of the graph traveres through the
-        graph in a depth first approach.
-        Retrieves output from an upstream operator at a particular output port.
-        Sets the right input to a downstrem operator at the right input port.
-        Executes the operators.
+        This method executes operators of the graph in topological order:
+
+        - If a node (operator) is a root node, its input is treated as a data path (DataPath, IOType.DISK).
```

Comment:
Multiple inputs supported?

Reply:
It is not supported yet and we have an issue regarding it https://github.com/Project-MONAI/monai-deploy-app-sdk/issues/87

All root nodes would be able to access only one input.